File: java/src/main/java/mx/iteso/ut/TortillaHarina.java
package mx.iteso.ut;

/**
 * A class by a tortillero.
 */
public class TortillaHarina implements Tortilla {

    /**
     * is it toasted? the people exclaimed.
     * @return false
     */
    public boolean isToasted() {
        return false;
    }

    /**
     * gtt.
     * @return 0
     */
    public int getToastTemperature() {
        return 0;
    }

    /**
     * gct.
     * @return 0
     */
    public int getCurrentTemperature() {
        return 0;
    }

    /**
     * sct.
     * @param temp temp
     */
    public void setCurrentTemperature(
            final int temp
    ) {

    }

    /**
     * t.
     * @param toasted tasted
     */
    public void toast(
            final boolean toasted
    ) {

    }
}

File: csharp/unittest-practice/Clases/TortillaHarina.cs
﻿

namespace unittestpractice.Clases
{
    public class TortillaHarina : ITortilla
    {
        private bool _toasted;
        private int _temperature;
        private const int Toasting = 40;

        public bool isToasted()
        {
            return _toasted;
        }

        public int getCurrentTemperature()
        {
            return _temperature;
        }

        public int getToastTemperature()
        {
            return Toasting;
        }

        public void setCurrentTemperature(int temp)
        {
            _temperature = temp;
        }

        public void toast(bool toasted)
        {
            _toasted = toasted;
        }
    }

}
